packages/cli/src/cmds/validator/options.ts

```diff
@@ -260,7 +260,7 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
   "builder.boostFactor": {
     type: "string",
     description:
-      "Percentage multiplier the block producing beacon node must apply to boost (>100) or dampen (<100) builder block value for selection against execution block. The multiplier is ignored if `--builder.selection` is set to anything other than `default` or `maxprofit`",
```

Comment:
Since we have the `default` is for users that do not change the defaults it makes more sense to only apply manually set boost factor in case of `maxprofit` as is anyhow a setting for more advanced users.